src/ir/static_init.rs

```diff
@@ -422,7 +423,10 @@ impl Literal {
                     x, f
                 )),
             }),
-            Literal::Str(string) => Ok(string.into_boxed_slice()),
+            Literal::Str(_) => {
+                todo!("string literal")
```

Comment:
Maybe something like this?
```rust
Ok(self.literal_parser_iter(expr.location)
    .parse_string_raw()
    .unwrap()
    .into_boxed_slice())
```

Reply:
Problem is this is in `impl Literal` not `Compiler` so I'd have to pass in `&Compiler`. Not terribly inconvenient.

Reply:
I moved `into_bytes` into `impl Compiler` and it works. Hopefully that's not an issue in the eyes of the API.